demos/input/button/source/PlayState.hx

```diff
@@ -0,0 +1,23 @@
+package;
+
+import flixel.ui.FlxButton;
+import flixel.FlxG;
+import flixel.FlxState;
+import flixel.util.FlxColor;
+
+class PlayState extends FlxState
+{
+	var button:FlxButton;
+
+	override public function create()
+	{
+		super.create();
+
+		button = new FlxButton(120, 110, "Click me", onButtonClicked);
+		add(button);
+	}
+
+	function onButtonClicked() {
```

Comment:
Minor code formatting inconsistency here (tip: you can use haxe-formatter, which is built into VSCode).

Reply:
I must have messed with the formatter!